api_tests/src/actors/actor.ts

```diff
@@ -256,6 +256,46 @@ export class Actor {
         );
     }
 
+    public async initOrderbookTest(
+        swapUrl: string,
+        network: string,
+        chainId: number,
+        btcAmount: string,
+        erc20Amount: string,
+        erc20TokenContract: string
+    ) {
+        this.alphaLedger = {
+            name: LedgerKind.Bitcoin,
```

Comment:
Please tell me this was a mistake and that you understand that you are doing a herc20-hbit swap and that the alpha ledger is Ethereum.